src/components/Badge/Badge.tsx

```diff
@@ -59,7 +60,9 @@ const defaultProps = {
 	type: Type.filled,
 };
 
-export const Badge = (props: IBadgeProps): React.ReactElement => {
+export const Badge: FC<IBadgeProps> = (
+	props: IBadgeProps
+): React.ReactElement => {
 	const { className, kind, type, children, ...passThroughs } = props;
```

Comment:
I've seen this written as...

```ts
export const Badge: FC<IBadgeProps> = ({ className, kind, type, children, ...passThroughs }) => {
```

which is basically the same but kinda looks cleaner in my opinion. 

No need to change it, but I thought I'd mention in case that looks appealing to you 😄

Reply:
Thanks for the suggestion. I think that's easier to read. Maybe it makes sense to do that rather than destructure the props on line 66.